dask/bytes/local.py

```diff
@@ -134,3 +145,55 @@ def getsize(path, compression=None):
             result = g.tell()
             g.close()
         return result
+
+
+class LocalFileSystem(object):
+    """API spec for the methods a filesystem
+
+    A filesystem must provide these methods, if it is to be registered as
+    a backend for dask.
+
+    Implementation for local disc"""
+    sep = os.sep
+
+    def __init__(self, **storage_options):
+        """
+        Parameters
+        ----------
+        storage_options: key-value
+            May be credentials, or other configuration specific to the backend.
+        """
+        # no configuration necessary
+        pass
+
+    def glob(self, path):
+        """For a template path, return matching files"""
+        return glob(path)
+
+    def mkdirs(self, path):
+        """Make any intermediate directories to make path writable"""
+        return os.makedirs(path, exist_ok=True)
+
+    def open(self, path, mode='rb', **kwargs):
+        """Make a file-like object
+
+        Parameters
+        ----------
+        mode: string
+            normally "rb", "wb" or "ab" or other.
+        kwargs: key-value
+            Any other parameters, such as buffer size. May be better to set
+            these on the filesystem instance, to apply to all files created by
+            it. Not used for local.
+        """
+        return open(path, mode=mode)
+
+    def ukey(self, path):
+        """Unique identifier, so we can tell if a file changed"""
+        return tokenize(path, os.stat(path).st_mtime)
```

Comment:
Other times we use `getmtime` or something I think.  Is there a difference here?